### PR TITLE
LPS 122347

### PR DESCRIPTION
**Pull request by 4lejandrito**

### CODE REVIEW

**Comment by 4lejandrito** [comment]
ci:test:sf

**Comment by 4lejandrito** [comment]
ci:test:relevant

**Comment by 4lejandrito** [comment]
ci:test:stable

File: modules/apps/document-library/document-library-external-video/src/main/java/com/liferay/document/library/external/video/internal/resolver/ExternalVideoResolver.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+				return externalVideo;
+			}
+		}
+
+		return null;
+	}
+
+	@Activate
+	protected void activate(BundleContext bundleContext) {
+		_embedProviders = ServiceTrackerListFactory.open(
```
Comment (adolfopa) [inline modules/apps/document-library/document-library-external-video/src/main/java/com/liferay/document/library/external/video/internal/resolver/ExternalVideoResolver.java:48]: Shouldn't this be named "_externalVideoProviders"?

File: modules/apps/document-library/document-library-external-video/src/main/java/com/liferay/document/library/external/video/internal/constants/ExternalVideoWebKeys.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ */
+
+package com.liferay.document.library.external.video.internal.constants;
+
+/**
+ * @author Alejandro Tardín
+ */
+public class ExternalVideoWebKeys {
+
+	public static final String EMBEDDED_HTML = "EMBED_HTML";
```
Comment (adolfopa) [inline modules/apps/document-library/document-library-external-video/src/main/java/com/liferay/document/library/external/video/internal/constants/ExternalVideoWebKeys.java:22]: Shouldn't this be named "EMBED_HTML"? Or alternatively, rename the constant value to "EMBEDDED_HTML".

File: modules/apps/document-library/document-library-external-video/src/main/java/com/liferay/document/library/external/video/internal/resolver/ExternalVideoResolver.java (hunk 1 above)
Comment (4lejandrito) [reply modules/apps/document-library/document-library-external-video/src/main/java/com/liferay/document/library/external/video/internal/resolver/ExternalVideoResolver.java:48]: yeah 😓 I renamed it 1000 times and that one shows a previous naming.

**Comment by 4lejandrito** [comment]
ci:forward

**Comment by 4lejandrito** [comment]
ci:forward

**Comment by 4lejandrito** [comment]
ci:forward

**Pull request closed without merging**